src/clojure/clj_ldap/client.clj

```diff
@@ -99,19 +99,22 @@
     (when (true? verify-host?) (.setSSLSocketVerifier opt (HostNameSSLSocketVerifier. (true? wildcard-host?))))
     opt))
 
-(defn- create-trust-manager
-  "If the trust-store is truthy, returns a TrustStoreTrustManager created with
-  it; otherwise, returns a TrustAllTrustManager."
-  [trust-store]
-  (if trust-store
-    (TrustStoreTrustManager. trust-store)
-    (TrustAllTrustManager.)))
+(defn- create-ssl-util
+  "If the trust-manager is truthy, returns a SSLUtil created with
+  it; otherwise, if trust-store is truthy, returns a SSLUtil created with
+  it. If both are falsy, returns a SSLUtil created with a TrustAllTrustManager."
+  [trust-managers trust-store]
+  (if trust-managers
+      (Utils/trustManagersToSSLUtil trust-managers)
```

Comment:
Nit: indentation is off here (should be on the same level as the next if). Cursive thinks it should look like this:

```
(defn- create-ssl-util
  "If the trust-manager is truthy, returns a SSLUtil created with
  it; otherwise, if trust-store is truthy, returns a SSLUtil created with
  it. If both are falsy, returns a SSLUtil created with a TrustAllTrustManager."
  [trust-managers trust-store]
  (if trust-managers
    (Utils/trustManagersToSSLUtil trust-managers)
    (if trust-store
      (Utils/trustStoreToSSLUtil trust-store)
      (Utils/trustManagerToSSLUtil
        (TrustAllTrustManager.)))))
```

Reply:
pushed an update to fix